test/functional/feature_bip68_sequence.py

```diff
@@ -91,7 +91,7 @@ def test_disable_flag(self):
         utxo = self.wallet.send_self_transfer(from_node=self.nodes[0])["new_utxo"]
 
         tx1 = CTransaction()
-        value = int((utxo["value"] - self.relayfee) * COIN)
+        value = btc_to_sat(utxo["value"] - self.relayfee)
```

Comment:
I think it is a fair enough critique, although I don't fully agree that isn't enforceable;

PRs go through review, and the test_framework exists as a facilitator; It is part of the review process to see if we are adding repetitive code that could be abstracted away;

Many parts of the test_framework could arguably be inlined; Even the `COIN` constant itself could be removed, and replaced with just 1e8, since that is bounded to never change;

Out of curiosity: would a class `Sats/Coin/Bitcoin` with constructors for str, int and decimal, that has these two methods to represent the value in different unit scales be any better?

Or do you believe the concept is flawed in itself?

On the defaults: That is already a problem on the test suite - there is some interactions of float and decimal, and by definition all operations should be in sats/integer and just displayed as BTC when it makes sense; I could tackle that, but I believe it would make the most sense to be in a different issue/PR?